src/main/java/com/nylas/Event.java

```diff
@@ -149,6 +158,7 @@ public void setRecurrence(Recurrence recurrence) {
 		Maps.putIfNotNull(params, "location", getLocation());
 		Maps.putIfNotNull(params, "participants", getParticipants());
 		Maps.putIfNotNull(params, "busy", getBusy());
+		Maps.putIfNotNull(params, "metadata", getMetadata());
```

Comment:
please do not merge until we decide on the plan for this issue

Reply:
Good idea. Holding off.

Reply:
The issue has been fixed and deployed in Prod. Example code reflects the current working state.